src/werkzeug/middleware/shared_data.py

```diff
@@ -11,6 +11,7 @@
 
 from __future__ import annotations
 
+import collections.abc
```

Comment:
Alias this to a shorter name with `import collections.abc as cabc`, that's what I've been using elsewhere.